package.json

```diff
@@ -104,6 +105,7 @@
         "babel-jest": "^27.3.1",
         "dotenv": "^8.2.0",
         "react": "^17.0.2",
+        "react-beautiful-dnd": "^13.1.1",
```

Comment:
if you remove it from the `main package.json` and only add it to `package.json in /trader`, there won't be any problem with `package-lock` file :)

Reply:
Yeah, I moved it to the correct place, however, the lock file didn't rebuild correctly) I see why from your other comment. Very bad process we're following.

Reply:
100%